pkg/cli/advisory_discover.go

```diff
@@ -0,0 +1,40 @@
+package cli
+
+import (
+	"net/http"
+
+	"github.com/spf13/cobra"
+	"github.com/wolfi-dev/wolfictl/pkg/advisory"
+	"github.com/wolfi-dev/wolfictl/pkg/vuln/sftracker"
+)
+
+func AdvisoryDiscover() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "discover",
+		Short:         "search for new potential vulnerabilities and create advisories for them",
+		SilenceErrors: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			index, err := newConfigIndexFromArgs(args...)
+			if err != nil {
+				return err
+			}
+
+			secfixesTracker := sftracker.New(
+				"secfixes-tracker-nd2dq3gc7a-uk.a.run.app",
```

Comment:
constant/option plz

Reply:
Great thinking, added in 899f590ad2386bdb0c8f43d2cb9bbd059842106d